scripts/services/setConfig.ts

```diff
@@ -0,0 +1,12 @@
+import { compat, types as T } from "../deps.ts";
+
+// deno-lint-ignore require-await
+export const setConfig: T.ExpectedExports.setConfig = async (
+  effects: T.Effects,
+  newConfig: T.Config,
+) => {
+  const dependsOnCln: { [key: string]: string[] } = !!newConfig?.advanced.plugins.rest ? { "c-lightning": [] } : {};
```

Comment:
```suggestion
  const dependsOnCln: { [key: string]: string[] } = newConfig?.advanced.plugins.rest ? { "c-lightning": [] } : {};
```

Reply:
also, do we really have no pointers to CLN? is there no auth information?

Reply:
nope, just a macaroon in a pointer volume